vispy/visuals/image.py

```diff
@@ -126,6 +126,53 @@
 _null_color_transform = 'vec4 pass(vec4 color) { return color; }'
 _c2l_red = 'float cmap(vec4 color) { return color.r; }'
 
+_polar_transform = """
+    vec4 polar_transform_map(vec4 pos) {
+        float PI = 3.14159265358979323846;
+        float PI2 = PI * 2;
```

Comment:
Nitpick, this int literal, and some others below, look better as float literals, since they all interact with floats.

Reply:
I'll fix this.

Reply:
Done